plans/example-browser/playwright-runner.js

```diff
@@ -0,0 +1,41 @@
+const { chromium, firefox } = require('playwright')
+
+module.exports = (module) => {
+  const testFn = require(module)
+  return async (runenv, client) => {
+    runenv.recordStart()
+    await runTestFn(runenv, client, testFn)
+    runenv.recordSuccess()
+  }
+}
+
+async function runTestFn (runenv, client, fn) {
+  let browser
+  let result
+
+  try {
+    runenv.recordMessage('playwright: launching browser and opening new page')
+    if (/firefox/i.exec(runenv.testInstanceParams.browser)) {
```

Comment:
Is this the standard way of checking which browser to open? 
I would prefer a simple string check, and if the user inputs something like "opera", we should return an error.

Reply:
No, not standard. Just my way of integrating that configurability into testground using a simple regex (to not have to worry about surrounding whitespaces or case sensitivity). So there are two things here:

1. require an exact string (be it case insensitive or not)
2. also expect for chromium an exact string, which would be the default

I can apply both things. Which I'll assume for now is what you want.

Reply:
Ignoring whitespaces or case sensitivity sounds like a good idea; I would apply both points.

Reply:
Both are for me in this version,

In future iterations, let's favor exact string matches when we can: It's hard to plan for how a parameter might evolve and what edge cases might bite us in the future. (remember windows 9? :)

Reply:
Resolved as an exact string, as requested.